README.md

```diff
@@ -208,6 +208,32 @@ const getMarkdown = () => {
 - *Ruby* - [`<<~` Heredocs](https://www.rubyguides.com/2018/11/ruby-heredoc/)
 - *Swift* - [multiline string literals](https://github.com/mmkal/proposal-multi-backtick-templates) using triple-quotes - strips margin based on whitespace before closing delimiter
 
+## Syntax Alternatives Considered
+
+Some potential alternatives to the multi-backtick syntax:
+
+- A built-in runtime method along the lines of the `dedent` library, e.g. `String.dedent`:
+  - Pros:
+    - Easier to implement
+    - No new syntax
+  - Cons:
+    - Non-zero runtime impact (though less than a js library)
+    - Unclear how/whether it could work with tagged template literals
```

Comment:
I created https://github.com/jridgewell/string-dedent to flesh out the API. It would work with tagged templates.

Reply:
Ok, I can replace this one with the isTemplateObject issue you mentioned.